src/filler/handlers/neftymarket/processors/auctions.ts

```diff
@@ -153,7 +153,7 @@ export function auctionProcessor(core: NeftyMarketHandler, processor: DataProces
         updated_at_block: block.block_num,
         updated_at_time: eosioTimestampToDate(block.timestamp).getTime(),
         claimed_by_buyer: auctionType === AuctionType.DUTCH || (buyNowPrice > 0 && bidAmount >= buyNowPrice),
-        claimed_by_seller: auctionType === AuctionType.DUTCH || (buyNowPrice > 0 && bidAmount >= buyNowPrice),
+        claimed_by_seller: false,
```

Comment:
Why did this change?

Reply:
Hmm ok, maybe we should change the contract

Reply:
I've changed the contract https://github.com/neftyblocks/nefty-market/pull/3